metals/src/main/scala/scala/meta/internal/metals/callHierarchy/Demo.scala

```diff
@@ -0,0 +1,27 @@
+package scala.meta.internal.metals.callHierarchy
```

Comment:
Looks like this was added by mistake during tests? Could you remove it?

Reply:
agrh...